<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>piclonedialog</class>
 <widget class="QDialog" name="piclonedialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clone SD Card</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_from">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy From Device:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="from_cb">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>231</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="to_cb">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>231</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_to">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Copy To Device:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="resize_cb">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>401</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Resize Last Partition</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>piclonedialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>piclonedialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
